src/v2/Apps/Settings/SettingsApp.tsx

```diff
@@ -41,6 +41,11 @@ const tabs = [
 ]
 
 const SettingsApp: React.FC<SettingsAppProps> = ({ me, children }) => {
+  if (!me) {
+    // TODO: `@principalField` isn't working here and can't get `onServerSideRender` to work
```

Comment:
So this is weird. `@principalField` works in a raw query but for some reason it's not working to 403 in the application.

I also couldn't get `onServerSideRender` to execute at all.